core/src/commonMain/kotlin/behavior/GuildBehavior.kt

```diff
@@ -824,6 +824,29 @@ public suspend inline fun GuildBehavior.createNewsChannel(
     return Channel.from(data, kord) as NewsChannel
 }
 
+/**
+ * Requests to create a new stage channel.
+ *
+ * @return The created [StageChannel].
+ *
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun GuildBehavior.createStageChannel(
+    name: String,
+    builder: StageChannelCreateBuilder.() -> Unit = {}
+): StageChannel {
+    contract {
+        callsInPlace(builder, EXACTLY_ONCE)
+    }
+    val response = kord.rest.guild.createStageChannel(id, name) {
+        builder()
+        parentId = id
+    }
```

Comment:
setting the `parentId` to the guild id is also wrong, just noticed after previous comment

Reply:
Ah yep, I see it

Reply:
Fixed again